<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="gramatica">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gramática</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tabelaParse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>701</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela de Parsing LL(1)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nomeArquivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="lerDoArquivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ler do arquivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reconhecer">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reconhecer sentença</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sentenca">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="possuiRE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Possui RE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fatorada">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fatorada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="condicao3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3 condição</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="first">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="follow">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="firstNT">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Follow</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First-NT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salvar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OK">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
